src/config.c

```diff
@@ -344,7 +459,24 @@ CONFIG_SETTER(setFrisoINI) {
   RETURN_STATUS(acrc);
 }
 CONFIG_GETTER(getFrisoINI) {
-  return config->frisoIni ? sdsnew(config->frisoIni) : NULL;
+  if (config->frisoIni && strlen(config->frisoIni) > 0) {
+    return sdsnew(config->frisoIni);
+  } else {
+    return NULL;
+  }
+}
+
+// friso-ini
+RedisModuleString * get_friso_ini(const char *name, void *privdata) {
+  char *str = *(char **)privdata;
+  if (str) {
+    if (config_friso_ini) {
```

Comment:
Why do you need `config_friso_ini` as a global variable?
Can't you just return a redis module string and have redis free it?